tests/integration/test_checker.py

```diff
@@ -1,9 +1,41 @@
+import errno
+import os
+import shutil
+import tempfile
 import unittest
-from unittest.mock import patch
-from unittest.mock import mock_open
-
+from unittest.mock import mock_open, patch
 
 from mavis import checker
+from mavis.constants import COMPLETE_STAMP, SUBCOMMAND
+
+MOCK_GENOME = 'mock-A36971'
+MOCK_TRANS = 'mock-A47933'
+ERROR_MESSAGE = """Traceback (most recent call last):
+  File "/home/dpaulino/gitrepo/mavis/venv/bin/mavis_run.py", line 6, in <module>
```

Comment:
Is this test error message really intended to have paths to Daniel's home, /home/dpaulino/gitrepo/mavis/bin/mavis_run.py, and /projects/tumour_char/analysis_scripts/python/centos06/python-3.6.0/lib/python3.6/subprocess.py?